tests/test_bear_deploy.py

```diff
@@ -57,7 +57,7 @@ def test_bear_deploy():
     # test that the institution deploys a BearStore
     assert_in("New fac: BearStore", s)
     # test that the first agents exist with right minimum production.
-    agents = re.compile('Agent \d+ 8\.0')
+    agents = re.compile('Agent \d+ 10\.0')
```

Comment:
why change this test in this way? (Sorry to be picky, but changes in the tests are usually a little bit of a no-no for me.) Adding tests is always fine, but altering or removing tests can allow changes to introduce backwards incompatibility, loss of functionality, etc.